app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.js

```diff
@@ -94,6 +94,15 @@ module.exports = angular
       });
     };
 
+    this.restartDetails = () => {
```

Comment:
normally no, you don't do this.  it would be better to execute this once in the controller and store the result in a field on `this` in the component controller.

that said, since you've one-time bound the results in the template, angular won't execute the function or watch the return value after the function returns something that's not `undefined`.